hermes-consumers/src/test/java/pl/allegro/tech/hermes/consumers/consumer/rate/maxrate/HierarchicalCacheMaxRateRegistryTest.java

```diff
@@ -32,10 +32,10 @@ public class HierarchicalCacheMaxRateRegistryTest extends ZookeeperBaseTest {
     private final MaxRatePathSerializer pathSerializer = new MaxRatePathSerializer();
     private final SubscriptionsCache subscriptionsCache = mock(SubscriptionsCache.class);
     private final ConfigFactory configFactory = new MutableConfigFactory();
-    private final String cluster = configFactory.getStringProperty(Configs.KAFKA_CLUSTER_NAME);
+    private final String cluster = "primary-dc";
```

Comment:
why this property is not in the `application-intergation.yml`?

Reply:
now, properties aren't loaded from the application-integration.yml file. 😐 If I'm not wrong we have to rewrite some part of tests to use spring context (application.yml files). I don't want to do this in this PR